app/models/service.rb

```diff
@@ -30,7 +30,6 @@ class Service < ApplicationRecord
   after_save :deleted_without_state_machine
 
   before_destroy :stop_destroy_if_last_or_default
-  before_destroy :destroy_features
```

Comment:
Hmmmm ok but then you also need to change this line

https://github.com/3scale/porta/blob/44d0ad2774655821077dbf772da0404abae38d14/app/models/service.rb#L87

`dependent: :destroy` otherwise it will be `:delete_all`